config/filter.d/apache-auth.conf

```diff
@@ -12,10 +12,10 @@ before = apache-common.conf
 
 failregex = ^%(_apache_error_client)s (AH(01797|01630): )?client denied by server configuration: (uri )?\S*(, referer: \S+)?\s*$
             ^%(_apache_error_client)s (AH01617: )?user .*? authentication failure for "\S*": Password Mismatch(, referer: \S+)?$
-            ^%(_apache_error_client)s (AH01618: )?user .*? not found(: )?\S*(, referer: \S+)?\s*$
-            ^%(_apache_error_client)s (AH01614: )?client used wrong authentication scheme: \S*(, referer: \S+)?\s*$
+            ^%(_apache_error_client)s (AH01618: )?user .*? not found(: )?.*(, referer: \S+)?\s*$
```

Comment:
> do you think we should use `\S*` to also catch empty user ...

I don't think so, because that depends. For example if by the handshake, before the client send authentication data, the web-server (or application) will produce 401 (and failure in log in sense of fail2ban), it is also counted. So the client should make `maxretry` of such "failures" - it goes banned.

This question is thus not really answerable as long as too few info... But rather - no, because makes no sence.